Source/Core/DolphinWX/Debugger/MemoryWindow.h

```diff
@@ -35,7 +35,9 @@ class CMemoryWindow : public wxPanel
   DECLARE_EVENT_TABLE()
 
   void OnDataTypeChanged(wxCommandEvent& event);
-  void OnSearch(wxCommandEvent& event);
+  void OnFindNext(wxCommandEvent& event);
+  void OnFindPrevious(wxCommandEvent& event);
+  void Search(bool find_next = true);
```

Comment:
This shouldn't be intermixed with all of the event functions.

Also please don't use default arguments for a function that's essentially used equally in both cases. Furthermore, an `enum class` is more appropriate here with regards to the parameter. `Search(true)` and `Search(false)` are pretty ambiguous as to what they do from a self-documenting point of view.